site/book/04-using-functions/02-imperative-function-execution.md

```diff
@@ -136,7 +136,7 @@ For example, the following will set the labels on all resources in wordpress pac
 the `Kptfile`:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/set-label:v0.1 --include-meta-resources wordpress -- label_name=color label_value=orange
+$ kpt fn eval --image gcr.io/kpt-fn/set-label:v0.1 --include-meta-resources wordpress -- color=orange
```

Comment:
I would still keep two labels here, it's good to show that.  how about color=orange org=shipping

Reply:
Done